android/app/src/main/java/org/fossasia/openevent/adapters/BaseRVAdapter.java

```diff
@@ -59,6 +59,9 @@ public void clear() {
         notifyDataSetChanged();
     }
 
+    /**
+     * Return size of dataList.
+     */
```

Comment:
Commenting is good, but explaining pretty obvious code is anti-pattern. Please remove :smile: